deployments/devel/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # This Dockerfile is also used to define the golang version used in this project
 # This allows dependabot to manage this version in addition to other images.
-FROM golang:1.23.1
+FROM golang:1.23.8
```

Comment:
Since we're still on `main`, this is done by https://github.com/NVIDIA/k8s-dra-driver-gpu/pull/312 I'm OK to do this here.

Reply:
> this is done by https://github.com/NVIDIA/k8s-dra-driver-gpu/pull/312 I'm OK to do this here.

Right. This PR here is slightly more conservative as it stays within 1.23.x

Reply:
Yes, in other projects we configure dependabot to only allow patch updates on `release-*` branches. This would have skipped these minor version updates.